app/Keycloak/KeycloakServiceProvider.php

```diff
@@ -48,5 +54,20 @@ public function register(): void
                 Uuid::fromString(config('keycloak.scope.widgets_id')),
             );
         });
+
+        $this->app->singleton(KeycloakClientRepository::class, function () {
+            return $this->app->get(EloquentKeycloakClientRepository::class);
+        });
+
+        $this->applyEvents();
+    }
+
+    private function applyEvents(): void
+    {
+        if (!$this->app->get(Config::class)->isEnabled) {
```

Comment:
Nit: It is also possible to create the Config class on the stack on line 40 and then pass it in here. But both are fine.